src/types/twitch.d.ts

```diff
@@ -774,6 +774,7 @@ declare module Twitch {
 	export type VideoPlayerComponent = ReactExtended.WritableComponent<{
 		containerRef: HTMLDivElement;
 		mediaPlayerInstance: MediaPlayerInstance;
+		children: any;
```

Comment:
i added this because the linter in vscode gave me this error
```
Property 'children' does not exist on type 'Readonly<{ containerRef: HTMLDivElement; mediaPlayerInstance: MediaPlayerInstance; }> & Writeable<Readonly<{ containerRef: HTMLDivElement; mediaPlayerInstance: MediaPlayerInstance; }>>'.ts(2339)

```